ksql-rest-app/src/main/java/io/confluent/ksql/rest/client/KsqlRestClient.java

```diff
@@ -344,14 +347,19 @@ public Object unsetProperty(final String property) {
     return propertiesMap;
   }
 
-  private static URI parseServerAddress(final String serverAddress) {
+  private static URI[] parseServerAddress(final String serverAddress) {
```

Comment:
Go with `List<URI>` over raw arrays.  Also, try to keep state immutable. 

If you leave the old `parseServerAddress` as it is, then you can use it from a new `parseServerAddresses` method using Java 8 streams, like this:

```java
private static List<URI> parseServerAddresses(final String serverAddresses) {
   return ImmutableList.copyOf(
      Arrays.stream(serverAddress.split(","))
         .map(String::trim)
         .map(KsqlRestClient::parseServerAddress)
         .collect(Collectors.toList()));
}

private static URI parseServerAddress(final String serverAddress) {
   // as it was
}
```

Note the use of `ImmutableList` to ensure the state is immutable.